src/deadline/client/cli/_groups/manifest_group.py

```diff
@@ -0,0 +1,199 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+All the `deadline manifest` commands:
+    * snapshot
+    * upload
+    * diff
+    * download
+"""
+from __future__ import annotations
+
+import dataclasses
+import os
+from typing import List
+import click
+
+from deadline.job_attachments.api.manifest import (
+    _manifest_snapshot,
+)
+
+from ...exceptions import NonValidInputError
+from .._common import _handle_error
+from .click_logger import ClickLogger
+
+
+@click.group(name="manifest")
+@_handle_error
+def cli_manifest():
+    """
+    Commands to work with AWS Deadline Cloud Job Attachments.
+    """
+
+
+@cli_manifest.command(
+    name="snapshot",
+    help="BETA - Generates a snapshot of files in a directory root as a Job Attachment Manifest.",
+)
+@click.option("--root", required=True, help="The root directory to snapshot. ")
+@click.option(
+    "-d",
+    "--destination",
+    default=None,
+    help="Destination directory where manifest is created. Defaults to the manifest root directory.",
+)
+@click.option(
+    "-n", "--name",
+    default=None,
+    help="Name of the manifest. A timestamp is added YYYY-MM-DD-HH-MM-SS for versioning.",
+)
+@click.option(
+    "-i",
+    "--include",
+    default=None,
+    help="Glob syntax of files and directories to include in the manifest. Can be provided multiple times.",
+)
+@click.option(
+    "-e",
+    "--exclude",
+    default=None,
+    help="Glob syntax of files and directories to exclude in the manifest. Can be provided multiple times.",
+)
+@click.option(
+    "-ie",
+    "--include-exclude-config",
+    default=None,
+    help="Include and exclude config of files and directories to include and exclude. Can be a json file or json string",
+)
+@click.option("--diff", default=None, help="File Path to Asset Manifest to diff against.")
+@click.option("--json", default=None, is_flag=True, help="Output is printed as JSON for scripting")
```

Comment:
super nit - inconsistent with `.` in the end.

Reply:
Good catch!